### ISSUE TITLE
Problems building with non-system libxml2

### ISSUE THREAD

**Issue by chrispbradley** [OpenCMISS/manage#100]
If you don't have libxml2 installed then the build system should build it. Trouble is it doesn't give the correct information about requiring zlib as well and so there are link errors.

**Comment by hsorby** [comment]
There is not enough information here to make any suggestions.  All tested environments are currently having no issues with libxml2.

**Comment by chrispbradley** [comment]
Yes there is, this is the libxml2 zlib problem that you know about. It is just here as an official issue. If you delete the libxml2-dev system package then it will try to build libxml2 but run into problems because it does not pick up the zlib dependencies. Problems with module/config mode.
